alicloud/data_source_alicloud_ram_account_alias.go

```diff
@@ -26,13 +26,15 @@ func dataSourceAlicloudRamAccountAlias() *schema.Resource {
 func dataSourceAlicloudRamAccountAliasRead(d *schema.ResourceData, meta interface{}) error {
 	client := meta.(*connectivity.AliyunClient)
 
-	raw, err := client.WithRamClient(func(ramClient ram.RamClientInterface) (interface{}, error) {
-		return ramClient.GetAccountAlias()
+	raw, err := client.WithRamClient(func(ramClient *ram.Client) (interface{}, error) {
+		getRequest := ram.CreateGetAccountAliasRequest()
+		getRequest.Scheme= "HTTPS"
```

Comment:
schema should be set in the client init

In addition, please rename getRequest to request